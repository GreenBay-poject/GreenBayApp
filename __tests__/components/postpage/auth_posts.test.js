import React from 'react';
import renderer from 'react-test-renderer';
import AuthPosts from '../../../components/postpage/auth_posts';

describe('<AuthPosts />', () => {
    it('Sample AuthPosts', () => {
        const tree = renderer.create(<AuthPosts />).toJSON();
        console.log(tree)
        expect(tree).toMatchSnapshot();
        // expect(tree.children[0].type).toBe("Text");
        // expect(tree.children[1].type).toBe("TextInput");
        // expect(tree.children[2].type).toBe("TextInput");
        // expect(tree.children[3].type).toBe("View");
        // expect(tree.children[4].type).toBe("Text");


    });
});