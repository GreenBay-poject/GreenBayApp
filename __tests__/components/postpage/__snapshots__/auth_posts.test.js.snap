// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthPosts /> Sample AuthPosts 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 20,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#d7dbd8",
        "borderColor": "#153d37",
        "borderRadius": 10,
        "borderWidth": 1,
        "flex": 1,
        "height": 150,
        "marginHorizontal": 10,
        "opacity": 0.9,
        "padding": 10,
        "resizeMode": "cover",
        "width": 250,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#153d37",
          "fontSize": 20,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      Sorry !!!
    </Text>
    <Text
      style={
        Object {
          "color": "#153d37",
          "fontSize": 15,
          "textAlign": "center",
        }
      }
    >
      This Part Not Implemented yet.
    </Text>
  </View>
</View>
`;
